index.html

```diff
@@ -309,21 +316,30 @@ <h3>
 
           <ol>
             <li>
-Replace ":" with "/" in the method specific identifier to obtain the fully
-qualified domain name and optional path.
+Replace ":" with "/" in the method specific identifier, exclusive of any fragment,
```

Comment:
I'm not sure I understand the 'exclusive of any fragment' part -- what do you mean by that?

Reply:
I am referring to anything following `#` as the fragment.

In this example:
`did:web:example.com:alice#key9`

I refer to `key9` as the fragment.

We've not made any reference to fragments before now, and perhaps we can exclude that and simplify the language a bit further.

Reply:
I removed this in #47.